tests/platform_helper/fixtures/pipeline/platform-config-for-terraform.yml

```diff
@@ -1,13 +1,14 @@
+application: my-app
 environments:
-  - name: dev
+  dev:
     accounts:
       deploy:
         name: "dev-deploy-account"
         id: "1111111111"
       dns:
         name: "dev-dns-account"
         id: "2222222222"
-  - name: prod
+  prod:
```

Comment:
I'm wondering whether it's worth separating the definitive list of environments from the `environment_pipelines` configuration?

So we have...

```
application: my-app
environments:
  ...
environment_pipelines:
  ...
codebase_pipelines:
  ...
extensions:
  ...
```

Reply:
You resolved this, but there's no change or counter argument.

Reply:
Looks actioned in [demodjango-deploy](https://github.com/uktrade/demodjango-deploy/pull/112/files/de939a3ad22b9c55dca41ec18aed4dbd5d2272e4..7177c64420497302b4bb1dd50154ab77c970a3dd#diff-9daff048341e6ee1738e7774733bfd67520d000e5ba42701c4b6604dac9fa579) though?

Reply:
It is actioned. It won't show in this file as it doesn't need any pipeline config. If yo look in validation.py though you'll see the structure of the entire file which is now being validated. And the change to some keys in the copilot stuff to reference e.g. codebase_pipelines rather than codebases.